packages/core/src/amazonqGumby/index.ts

```diff
@@ -6,4 +6,7 @@
 export { activate } from './activation'
 export { default as DependencyVersions } from './models/dependencies'
 export { default as MessengerUtils } from './chat/controller/messenger/messengerUtils'
+export { GumbyController } from './chat/controller/controller'
+export { TabsStorage } from '../amazonq/webview/ui/storages/tabsStorage'
+export * as startTransformByQ from '../../src/codewhisperer/commands/startTransformByQ'
```

Comment:
I'm unable to run the tests in `packages/amazonq/test/e2e/amazonq/transformByQ.test.ts` by importing these things from the `core` module directly, as I get an import error when doing that. So instead I have to export them here and then (in the test file) import them from here.